### DISCUSSION TITLE
Query regarding GitHub Pro for Students

### DISCUSSION THREAD

**Discussion by simran-1213** [community/community#42569] [GitHub Education]
Hi there!

I had a doubt in mind regarding GitHub Pro along with Student Developer Pack for Students. If I am a bachelor's student (pursuing my undergrad) from university X and get the GitHub Pro activated on university X's student email. Then if I pursue my masters (post grad) from some other university Y. Will I still be able to use the GitHub Pro (since university mail would change)?

Thank you!

**Comment by jfullstackdev** [comment]
it's the right place you're posting but there is no assurance there will be a reply from the Staff

you can also try Support 

https://support.github.com/contact/account
